src/main/resources/static/css/login.css

```
@@ -0,0 +1,71 @@
+@media (min-width: 640px) {
+        .label-container {font-size: 24px;}
+        .input-field {min-width: 220px;}
+        .form-container {gap: 1.25rem;}
+        .invalid-state {font-size: 14px;}
+        .warning-state {font-size: 14px;}
+        .login-button {font-size: 20px;}
+	}
```

Consistent indenting, and prefer spaces within curly braces
```suggestion
    .label-container { font-size: 24px; }
    .input-field { min-width: 220px; }
    .form-container { gap: 1.25rem; }
    .invalid-state { font-size: 14px; }
    .warning-state { font-size: 14px; }
    .login-button { font-size: 20px; }
}
```